projects/backend/capi/articles.ts

```diff
@@ -292,7 +292,7 @@ const parseArticleResult = async (
 
 const isScheduledInNext30Days = (dateiso8601: string): boolean => {
     const date = new Date(dateiso8601)
-    const oneMonthAway = new Date(new Date().setDate(date.getDate() + 30))
+    const oneMonthAway = new Date(new Date().setDate(new Date().getDate() + 30))
```

Comment:
A potentially easier and more correct alternative, as a month may not be 30 days:

```ts
const date = new Date(dateiso8601)
const now = new Date();
const oneMonthAway = new Date(now).setMonth(now.getMonth() + 1))
return date < oneMonthAway;
```

Reply:
This would make `oneMonthAway` correct but make `isScheduledInNext30Days` untrue.

Reply:
Haha, indeed, I was more specifically trying to avoid the triple-nested `new Date()` by extracting `now`. Take 2:

```ts
const then = new Date(dateiso8601)
const now = new Date();
const thirtyDaysFromNow = new Date(now).setDate(now.getDate() + 30))
return then < thirtyDaysFromNow;
```